tests.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+import unittest
+from mtv_dl import _guess_series_details
+
+
+class TestSeasonEpisodeRegex(unittest.TestCase):
+
+    def _checkRegex(self, season: int, episode: int, title: str):
```

Comment:
I would prefer to go for a [doctest](https://docs.python.org/3/library/doctest.html) here.

Reply:
done